tests/terraform/get-tf-tests.py

```diff
@@ -0,0 +1,20 @@
+import sys
+
+import yaml
+
+
+def print_test_names(service):
+    with open("tests/terraform/terraform-tests.success.txt") as f:
+        dct = yaml.load(f, Loader=yaml.FullLoader)
+        tests = dct.get(service)
+        if len(tests) == 1:
+            print(tests[0])
+        else:
+            print('"(' + "|".join(tests) + ')"')
+
+
+if __name__ == "__main__":
+    if len(sys.argv) != 2:
+        print("")
```

Comment:
q: So if there is no parameter, or too many, then the output will be empty. Will this lead to all tests being executed?

Reply:
in case of empty string the build will error out.

Reply:
Ah, okay. In that case, it would be nice to exit here with a non-zero exit status to make that behavior explicit (and maybe also add a comment here in the code).

Reply:
I've added a new stage which checks if test are present or not.